index.html

```diff
@@ -0,0 +1,45 @@
+<script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script> 
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Homenaje a Fulano de Tal</title>
+    <link rel="stylesheet" type="text/css" href="styles.css">
+  </head>
+  
+  <body>
+    <div id="main">
+      
+      <div class="header">
+        <h2 id="title">Fulano de Tal</h2>
+      </div>
+      <div class="buttons" z-index: 2>
+        <div class="menubutton" id="button1">Inicio</div>
+        <div class="menubutton" id="button2">Retrato</div>
+        <div class="menubutton" id="button3">Biografía</div>
+        <div class="menubutton" id="button4">Enlaces</div>
+      </div>
+      
+      <div class="content">
+        <div id="img-div"><img id="image" src="https://i.ytimg.com/vi/cEJKfqU5efA/maxresdefault.jpg" alt="Fulano de Tal en una entrevista televisiva.">
```

Comment:
Always have the tags in a new line and indented from their parent container.
```suggestion
        <div id="img-div">
          <img id="image" src="https://i.ytimg.com/vi/cEJKfqU5efA/maxresdefault.jpg" alt="Fulano de Tal en una entrevista televisiva.">
```
The only exception to this are inline tags like `<em>` or `<strong>`. 

Closing tags should be in the same indentation level as the opening tag.